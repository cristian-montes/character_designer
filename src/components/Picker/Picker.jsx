import React from "react";

export default function Picker({head, setHead}){
    const headOptions = ['alien', 'cat', 'face', 'putin']

    return(
        <div>
            <label>
                Head
                <select value={head} onChange={(event) => setHead(event.target.value)}>
                    {headOptions.map((opt) =>(
                        <option key={opt}>{opt}</option>
                    ))}
                </select>
            </label>
        </div>
    )
}