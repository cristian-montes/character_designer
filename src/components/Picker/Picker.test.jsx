import {screen, render} from '@testing-library/react';
import Picker from './Picker';

it('it should render Picker Component', ()=> {
    const {container } = render(<Picker/>);
    const labelText = screen.getByLabelText(/Head/)


    expect(container).toMatchSnapshot();
    expect(labelText).toBeInTheDocument();
})