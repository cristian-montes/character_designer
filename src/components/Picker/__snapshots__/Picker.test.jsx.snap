// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render Picker Component 1`] = `
<div>
  <div>
    <label>
      Head
      <select>
        <option>
          alien
        </option>
        <option>
          cat
        </option>
        <option>
          face
        </option>
        <option>
          caiman
        </option>
      </select>
    </label>
    <label>
      Middle
      <select>
        <option>
          hoodie
        </option>
        <option>
          tmobile
        </option>
        <option>
          tshirt
        </option>
      </select>
    </label>
    <label>
      Bottom
      <select>
        <option>
          jeans
        </option>
        <option>
          skirt
        </option>
        <option>
          gold
        </option>
      </select>
    </label>
    <label>
      New Phrases
      <input
        type="text"
        value=""
      />
    </label>
    <button
      type="button"
    >
      Add Sayings
    </button>
  </div>
</div>
`;
